templates/documents/general_upload.html

```diff
@@ -33,6 +37,14 @@
 <input type="hidden" name="category_id" value="{$category_id}" />
 <input type="hidden" name="process" value="{$PROCESS}" />
 </form>
+<!-- Drag and drop uploader -->
```

Comment:
My biggest concern is not yet being able to get a working upload. We can deal with the HTML later, but I can't get basic functionality to work. @bradymiller have you had any success with this?